generator/lib/control.ml

```diff
@@ -419,56 +451,45 @@ let api =
           methods = [
             {
               Method.name = "probe";
-              description = "[probe uri]: looks for existing SRs on the storage device";
+              description = String.concat " " [
+                "[probe configuration]: can be used iteratively to narrow down configurations";
+                "to use with SR.create, or to find existing SRs on the backing storage";
+              ];
               inputs = [
-                uri;
+                configuration;
               ];
               outputs = [
                { Arg.name = "result";
-                 ty = 
-                   Type.Struct (
-                     ("srs", Type.(Array sr_stat), "SRs found on this storage device"), [
-                      "uris", Type.(Array (Basic String)), "Other possible URIs which may contain SRs"
-                   ]);
+                 ty = Type.(Array (Name "probe_result"));
                  description = "Contents of the storage device";
                }
               ];
             };
             {
               Method.name = "create";
-              description = "[create uuid uri name description configuration]: creates a fresh SR";
+              description = "[create uuid configuration name description]: creates a fresh SR";
```

Comment:
I wonder why we kept the `uuid` param here - we've removed it from `attach`, is it already in the `configuration`?